en/integrations/plugin-packs/procedures/applications-databases-elasticsearch.md

```diff
@@ -47,3 +150,50 @@ The following macros must be configured on host:
 | ELASTICSEARCHUSERNAME | The Elasticsearch instance username |               | centreon |
 | ELASTICSEARCHPASSWORD | The Elasticsearch instance password |               | centreon |
 
+## FAQ
+
+### Comment tester en ligne de commande et que signifient les options principales ?
+
+One the Plugin is installed, you can test it directly in the command line from your Centreon poller
+with the user *centreon-engine*:
+
+```bash
+su - centreon-engine\
+/usr/lib/centreon/plugins/centreon_elasticsearch.pl \ 
+--hostname=x.x.x.x \
+--port=9200 \
+--proto=http \
+--plugin=database::elasticsearch::restapi::plugin \
+--mode=node-statistics \
+--filter-name='Node 1'
+
+```
```

Comment:
Add credentials parameters as most of the user will probably have their Elastic API protected by Basic Auth and Password